params/protocol_params.go

```diff
@@ -160,6 +160,9 @@ const (
 	RefundQuotient        uint64 = 2
 	RefundQuotientEIP3529 uint64 = 5
 
+	BeaconRootStorageAddress byte   = 0x0B
```

Comment:
You _could_ do this, and later just cast to address
```suggestion
	BeaconRootStorageAddress = [20]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0xb}
```
Not necessarily better, I know

Reply:
```suggestion
	BeaconRootStorageAddress byte   = common.MustDecodeAddress("0x0000...000b")
```

Another idea here with a non-existent func `MustDecodeAddress`. Ultimately I think it is very important to have this param be an address because it will likely be a popular import by others.